internals/cli/cmd_enter.go

```diff
@@ -29,7 +30,8 @@ These subcommands are currently supported:
 (1) Services are not started.
 (2) No logs on stdout unless -v is used.
 (3) Services continue running after the subcommand succeeds.
-`
+`,
+	cmd.Personality.ProgramName)
```

Comment:
If my understanding is correct, a dynamic change of the `Personality` struct will not change the content of `longEnterHelp` if `longEnterHelp` has already been initialized. And that initialization is going to happen very early, when the file level content of `cmd_enter.go` is executed. I assume that happens when the package is imported, though I don't know Go well enough to be certain.

If all of that is true, then you effectively can't change the `Personality` after importing this one package (and any other with the same kind of definition). I think this data flow dependency would be really obscure.